<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" connectslotsbyname="false">
 <class>brightness_small</class>
 <widget class="QMainWindow" name="brightness_small">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget">
   <layout class="QGridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="comboDisplay"/>
    </item>
    <item row="0" column="1">
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="pushExpandBacklight">
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelSoftwareBrightness">
      <property name="text">
       <string>Software (xrandr) Brightness</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSlider" name="sliderBrightness">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="labelBrightness">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushSave">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelHardwareBacklight">
      <property name="text">
       <string>Hardware Backlight</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QSlider" name="sliderHardwareBacklight">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLabel" name="labelBacklight">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
